File: Solution/WebSolution/Monolithic/Models/Task.cs
namespace TFLPortal.Models;
public class Task
{
   public int Id{get;set;}
   public string? Title{get;set;}
   public string? TaskType{get;set;}
   public string? Description{get;set;}
   public DateTime AssignedOn{get;set;}
   public DateTime StartDate{get;set;}
   public DateTime DueDate{get;set;}
   public DateTime CreatedOn{get;set;}
   public string? Status{get;set;}

   public int AssignedTo{get;set;}  
   public int AssignedBy{get;set;}  
   public int ProjectId{get;set;}  
   public int SprintId{get;set;}
}

File: Solution/UISolution/TFLUIPortal/src/app/ProjectManager/Models/task.ts
export class Task {
    constructor(
      public id: number,
      public title: string,
      public taskType: string,
      public description: string,
      public projectId: number,
      public sprintId: number,
      public assignedTo: number,
      public assignedBy: number,
      public createdOn :string ,
      public assignedOn :string ,
      public startDate :string,
      public dueDate: string,
      public status :string
    ) {}
  }